Library/ViewModels/DiscoveryFiltersViewModel.swift

```diff
@@ -157,10 +157,32 @@ public final class DiscoveryFiltersViewModel: DiscoveryFiltersViewModelType,
 
     self.notifyDelegateOfSelectedRow
       .observeValues {
-        AppEnvironment.current.ksrAnalytics.trackDiscoveryModalSelectedFilter(params: $0.params)
+        AppEnvironment.current.ksrAnalytics.trackDiscoveryModalSelectedFilter(
+          params: $0.params,
+          discoveryFilterContext: self.getDiscoveryFilterContextFrom($0.params),
+          locationContext: .discoverOverlay
+        )
       }
   }
 
+  private func getDiscoveryFilterContextFrom(
```

Comment:
I think you can add the "from" as first parameter label of the function, `getDiscoveryFilterContext(from discoveryParams: DiscoveryParams ...)`

Reply:
I have a different opinion about this, but I'll change to your opinion as it seem conventional.